### PR TITLE
Miscellaneous cleanups

### PR DESCRIPTION
**Pull request by ddcc**
Fix checking element presence in map, eliminate explicit index counter
Use character instead of single-character string
Fix trailing whitespace

### CODE REVIEW

File: src/wasm.h [hunk 1]
```diff
@@ -1453,62 +1453,58 @@ class Module {
...
   FunctionType* getFunctionType(size_t i) { assert(i < functionTypes.size()); return functionTypes[i].get(); }
   Import* getImport(size_t i) { assert(i < imports.size()); return imports[i].get(); }
   Export* getExport(size_t i) { assert(i < exports.size()); return exports[i].get(); }
   Function* getFunction(size_t i) { assert(i < functions.size()); return functions[i].get(); }
 
-  FunctionType* getFunctionType(Name name) { assert(functionTypesMap[name]); return functionTypesMap[name]; }
-  Import* getImport(Name name) { assert(importsMap[name]); return importsMap[name]; }
-  Export* getExport(Name name) { assert(exportsMap[name]); return exportsMap[name]; }
-  Function* getFunction(Name name) { assert(functionsMap[name]); return functionsMap[name]; }
+  FunctionType* getFunctionType(Name name) { assert(functionTypesMap.find(name) != functionTypesMap.end()); return functionTypesMap[name]; }
```
Comment (dschuff) [inline src/wasm.h]: how about `assert(functionTypesMap.count(name))`

**Comment by dschuff** [comment]
LGTM, can you rebase the conflicts?

**Comment by ddcc** [comment]
Done.

**Comment by kripken** [comment]
Thanks!

**Pull request merged**